examples/local/fivenodenetwork/go.mod

```diff
@@ -0,0 +1,109 @@
+module github.com/ava-labs/avalanche-network-runner/examples/local/fivenodenetwork
```

Comment:
What's does having another `go.mod` / `go.sum` do here?

Reply:
Trying not to bring `github.com/faith/color` dependency to the main repo. If we are open to it, I can remove this workaround :)

Reply:
Ah, I see. Makes sense